### ISSUE TITLE
Inner keys unusable with case v20 and MT3 keycaps

### ISSUE THREAD

**Issue by adwaymalhotra** [zzeneg/pica40#2]
The inner index keycaps (T,Y,G,M on QWERTY) are hindered by the battery shroud. Perhaps we can make it about 1mm thinner.

Interestingly, with Durock T1s the keycaps just get stuck when pressed. With Gateron Blacks there is just some resistance, but the keycaps don't get stuck.

**Comment by zzeneg** [comment]
Hey, thank you for reporting it, I was not able to test that new wireless case myself. So sorry you have issues with it. There's not much space there unfortunately, I'd really like to fit 20mm battery in that shroud, otherwise it's too similar to the other battery placement. I moved the wall for 0.3mm, it's tight - but the keycap should fit. Please let me know it you can test it.

![image](https://github.com/zzeneg/pica40/assets/910255/a312b284-d19a-4751-826e-a6cbb2bed9b3)

Also, just FYI - you should be able to sand down the wall a bit to make it work.

**Comment by adwaymalhotra** [comment]
Unfortunately, I can't print it again. I had it printed from JLCPCB.

I did sand mine down a bit and now things work better. I think there might be differences in the switch design as well? I didn't face a problem with the gateron blacks that I have, just with the Durock T1s. Strange. 

Anyway, thanks for the change! If I build another one, I may try the new design out. 

Maybe you can add a warning in the readme? I think it's just the MT3 (and maybe other larger keycap profiles) that will face this problem.

**Comment by zzeneg** [comment]
Yeah, I kinda avoided adding that variation to readme at all, because it was not tested :) I think I'll order it myself then.
Definitely weird that different switches behave differently in that regard, maybe some variation in stem location. Thanks again!
